packages/experiments/src/components/SelectedItemsList/SelectedItemsList.tsx

```diff
@@ -1,16 +1,15 @@
 import * as React from 'react';
 
 import { ISelectedItemsList, ISelectedItemsListProps, BaseSelectedItem } from './SelectedItemsList.types';
-
 const _SelectedItemsList = <TItem extends BaseSelectedItem>(
   props: ISelectedItemsListProps<TItem>,
   ref: React.Ref<ISelectedItemsList<TItem>>,
 ) => {
-  const [items, updateItems] = React.useState(props.selectedItems || []);
+  const [items, updateItems] = React.useState(props.selectedItems || props.defaultSelectedItems || []);
   const renderedItems = React.useMemo(() => items, [items]);
 
   React.useEffect(() => {
-    updateItems(props.selectedItems || []);
+    updateItems(props.selectedItems || props.defaultSelectedItems || []);
   }, [props.selectedItems]);
```

Comment:
Do we need to add defaultSelectedItems here as well? Is this how we should be using this prop, or should it only really set the internal items state at the first render?